spec/appcenter_upload_spec.rb

```diff
@@ -6,8 +6,11 @@ def stub_check_app(status)
     )
 end
 
-def stub_create_app(status)
+def stub_create_app(status, app_name = "app", app_display_name = "app", app_os = "Android", app_platform = "Java")
```

Comment:
do we want the default `app_display_name` value to be different from `app_name` so that we can verify that `app_name` isn't used in place of `app_display_name`? same goes for test below

Reply:
Personally I'm not sure that we need that

cc @derpixeldan any thoughts?

Reply:
I think @mseyma was asking in the context of the test. I think that's a valid suggestion

Reply:
I think it's a valid suggestion for the test scenario.

Reply:
Oh, I got it. I already have a separate test especially for that, so it is covered